db/schema.rb

```diff
@@ -49,6 +49,8 @@
     t.boolean "approximate_conditional_end_date", default: false
     t.boolean "approximate_compensation_payment_date", default: false
     t.boolean "approximate_motoring_disqualification_end_date", default: false
+    t.string "compensation_payment_over_100"
+    t.string "sent_payment_receipt"
```

Comment:
Maybe we can rename this attribute to start also with compensation, as we try for all attributes related to some specific category to share part of the name so it's easy to refer and identify them.
Perhaps something like `compensation_receipt_sent` or similar. It's up to you!